lib/src/main/java/com/tubitv/media/views/TubiExoPlayerView.java

```diff
@@ -99,23 +100,23 @@ public TubiExoPlayerView(Context context, AttributeSet attrs, int defStyleAttr)
         boolean useController = true;
         int surfaceType = SURFACE_TYPE_SURFACE_VIEW;
         int resizeMode = AspectRatioFrameLayout.RESIZE_MODE_FIT;
-        int controllerShowTimeoutMs = PlaybackControlView.DEFAULT_SHOW_TIMEOUT_MS;
+        int controllerShowTimeoutMs = PlayerControlView.DEFAULT_SHOW_TIMEOUT_MS;
```

Comment:
i don't think we need this anymore, since we move this logic to the controllerView level